README.md

```diff
@@ -41,31 +41,38 @@ the following headers:
 - `x-debug-cache-key`: the computed cache key
 - `x-debug-referer`: the referer value, if any
 
-## Environment Variables
+## Environment variables
 
-Note: This section is not exhaustive.
+### `BOUNCER_ADDR`
 
-### `BOUNCER_PINNED_BASEURL_HTTP`
+Address on which to listen. The default value is: `:8888`
+
+### `BOUNCER_DB_DSN`
 
-If this is a unset, bouncer will randomly pick a healthy mirror from the
-database and return its base url. If this option is set, the mirror table is
-completely ignored and `BOUNCER_PINNED_BASEURL_HTTP` will be returned instead.
+Database DSN, see: https://github.com/go-sql-driver/mysql#dsn-data-source-name
+for more details about the format.
 
-This option acts on non ssl only products.
+### `BOUNCER_PIN_HTTPS_HEADER_NAME`
 
-Example: `BOUNCER_PINNED_BASEURL=download-sha1.cdn.mozilla.net/pub`
+When this flag is set and the request header value equals https, an HTTPS
+redirect will always be returned. The default value is: `X-Forwarded-Proto`,
+which means this feature is enabled by default.
+
+### `BOUNCER_PINNED_BASEURL_HTTP`
+
+Configure the base URL for non-SSL only products.
 
 ### `BOUNCER_PINNED_BASEURL_HTTPS`
 
-This option is exactly the same as `BOUNCER_PINNED_BASEURL_HTTP` but acts on ssl
-only products.
+Same as `BOUNCER_PINNED_BASEURL_HTTP` but SSL-only products.
```

Comment:
Not a new issue, but this doesn't seem to match what the code is actually doing, ever since https://github.com/mozilla-services/go-bouncer/commit/f07f348605ca535f1563f8275545ba4899908775?  I.e. we use BOUNCER_PINNED_BASEURL_HTTPS not just for SSL only products, but also for non-SSL only products if the request came in over https.

Reply:
Yeah, so what happens is the header may change the value of `sslOnly` before we get to decide which base URL to use.

That makes me think that we do not explicitly set this header in our config, though. Currently, things work well because we have ELB in front of our nginx + bouncer but we'll need to be sure this will be the case once on GCP.